src/ILLink.RoslynAnalyzer/RequiresISymbolExtensions.cs

```diff
@@ -43,5 +64,24 @@ private static bool IsInRequiresScope (this ISymbol member, string requiresAttri
 
 			return false;
 		}
+
+		/// <summary>
+		/// This method determines if the member has a Requires attribute and returns it in the variable requiresAttribute.
+		/// </summary>
+		/// <param name="member">Symbol of the member to search attribute.</param>
+		/// <param name="requiresAttribute">Output variable in case of matching Requires attribute.</param>
+		/// <returns>True if the member contains a Requires attribute; otherwise, returns false.</returns>
+		private static bool TryGetRequiresAttribute (ISymbol? member, string requiresAttribute, [NotNullWhen (returnValue: true)] out AttributeData? requiresAttributeData)
```

Comment:
Just curious - adding the nullcheck for `member` here is different from the existing code. I assume it's intentional and attempts to make the code more robust, right?

Reply:
Also curious if we need this. I'm not sure we ever need the added ability to be able to call this on a nullable `ISymbol?` variable, it looks like we only call this from places where `member` is not null.

Reply:
I think I agree - the nullable annotations on all of the callsites seem to suggest that the input can never be null.